ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClient.java

```diff
@@ -86,6 +101,16 @@ void createTopic(
    */
   Map<String, TopicDescription> describeTopics(Collection<String> topicNames);
 
+  /**
+   * Synchronous call to get a one topic's description.
+   *
+   * @param topicName topicName to describe
+   * @return the description if the topic exists, or else {@code Optional.empty()}
+   */
+  default TopicDescription describeTopic(String topicName) {
```

Comment:
Not being a fan of abstract classes, (I favour composition in almost all cases), I don't agree.

If we were in Scala this could be in a companion object method.  Here, it could be in a utility. But given this isn't a public API I'm not too worried about it being on the interface.  It's just a utility method really.

Reply:
I agree `validateTopicProperties` being public isn't really ideal... I was being lazy ;)

I've moved it into a util class as its purely functional.